pipeline/train/train.sh

```diff
@@ -102,6 +102,16 @@ if [[ -z ${USE_CPU+x} ]]; then
   extra_params+=('local')
 fi
 
+# Enable log & metrics publication only when the tracking script is available
+if ! command -v parse_tc_logs &> /dev/null
+then
+  echo "### Weight & Biases publication script is not available."
+  PARSER=cat
+else
+  echo "### Weight & Biases publication is available."
+  PARSER="parse_tc_logs --from-stream -v --wandb-project=${src}-${trg} --wandb-group ${model_type}.${training_type} --wandb-run-name $TASK_ID"
```

Comment:
@La0 naming should be consistent with the offline uploading. For group: `<experiment-name-from-training-config>_<taskcluster-group-id>`, for task: I think we parse the task label to get one of the following: finetune-student, student, teacher-1, teacher-2, backwards. Some examples:

<img width="567" alt="Screenshot 2024-04-22 at 2 26 36 PM" src="https://github.com/mozilla/firefox-translations-training/assets/2486505/895278a5-8e0f-46da-b132-2e4293b857e1">
<img width="566" alt="Screenshot 2024-04-22 at 2 22 10 PM" src="https://github.com/mozilla/firefox-translations-training/assets/2486505/45a07064-5e43-4538-931b-7b7011a999b8">

Reply:
The naming is now fully handled by the tracking Python code, detecting if we are running in Taskcluster, in CI or normal runs, and producing the names using the group publication algorithm.

I setup default values for CI as the configuration is not exposed through task group in that case

Reply:
- [x] The group name looks good
- [x] The graphs look good
- [ ] The run names are still not consistent with the offline publisher

It's important to make naming consistent to compare the same model across older and newer training. See the names in en-ca: https://wandb.ai/moz-translations/en-ca

Reply:
When this last issue is fixed, I'll run a train action to test how it displays graphs in real time and that it takes the values properly from the config

Reply:
Done in [37d227f](https://github.com/mozilla/firefox-translations-training/pull/501/commits/37d227f52bc7339e51d8fe6430357f1e2b47ca9a)